dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-dingtalk/src/main/java/org/apache/dolphinscheduler/plugin/alert/dingtalk/DingTalkAlertChannelFactory.java

```diff
@@ -64,6 +64,28 @@ public String name() {
                         .setRequired(false)
                         .build())
                 .build();
+        InputParam atMobilesParam = InputParam
+                .newBuilder(DingTalkParamsConstants.NAME_DING_TALK_AT_MOBILES, DingTalkParamsConstants.DING_TALK_AT_MOBILES)
+                .addValidate(Validate.newBuilder()
+                        .setRequired(false)
+                        .build())
+                .build();
+        InputParam atUserIdsParam = InputParam
+                .newBuilder(DingTalkParamsConstants.NAME_DING_TALK_AT_USERIDS, DingTalkParamsConstants.DING_TALK_AT_USERIDS)
+                .addValidate(Validate.newBuilder()
+                        .setRequired(false)
+                        .build())
+                .build();
```

Comment:
Does mention users by id work for dingtalk now?

Reply:
The document provided by the official website has this field
```
https://open.dingtalk.com/document/robots/custom-robot-access
```
I tested the mobile phone number, but I didn't find out what the userId of DingTalk is, so I didn't test the userId